tools/database_migration.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import requests
+from redo import retry
+from database.config import session
+from database.models import Testjobs
+
+# alertmanager server URL
+URL = "http://alertmanager.allizom.org/data/dump/?limit=%d&offset=%d"
+
+
+def migration(args):
+    limit = int(args.limit)
+    offset = 0
+    url = URL % (limit, offset)
+    response = retry(requests.get, args=(url, )).json()
+    datasets = response['result']
+    while len(datasets) > 0:
+        for data in datasets:
+            Testjob = Testjobs(data['slave'], data['result'], data['build_system_type'],
+                               data['duration'], data['platform'], data['buildtype'], data['testtype'],
+                               data['bugid'], data['branch'], data['revision'], data['date'],
+                               data['failure_classification'], data['failures'])
+            try:
+                session.add(Testjob)
+                session.commit()
+
+            except:
+                session.rollback()
+
+            finally:
+                session.close()
```

Comment:
do we need to close the session each time, or can we leave it open until the while loop is finished?